geeksay.js

```diff
@@ -98,9 +98,13 @@ const translations = {
     "equal": "=",
     "remove": "rm",
     "move": "mv",
-    "plus": "+",
     "random": "rand",
     "mathematics": "math",
+    "back":"ctrl+z",
+    "send":"POST",
+    "receive":"GET",
+    "receive":"pull",
```

Comment:
`translations` is an object, so duplication of keys is something invalid (you added two times: `receive`)

Reply:
First of all thank you so much for reviewing my PR, the suggested changes are done.